scripts/healthomics_wf/utils/validate_globals_docker_images.py

```diff
@@ -45,15 +51,6 @@ def file_md5(file_path: str) -> str:
     return file_md5
 
 
-def parse_wdl_for_images(wdl_file: Path) -> set:
```

Comment:
why not to use the `parse_wdl_for_s3_paths` from `localize_s3.py` in the same way?

Reply:
not sure I understand what you mean